manifests/n/Nostalgia-09/AeroChat/1.0.1-rc.3/Nostalgia-09.AeroChat.locale.en-US.yaml

```
@@ -0,0 +1,21 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Nostalgia-09.AeroChat
+PackageVersion: 1.0.1-rc.3
+PackageLocale: en-US
+Publisher: Nostalgia-09
+PublisherUrl: https://discord.gg/3XfKgDMFBS
+PublisherSupportUrl: https://github.com/Nostalgia-09/AeroChat/issues
+PackageName: AeroChat

```

```suggestion
PackageName: Windows Live Messenger
```
Should match control panel